backend/.env.ci

```diff
@@ -0,0 +1,4 @@
+RECORD_TYPES = ["wikipedia"]
+OUTPUT_FILE_NAMES = ["mini-wikipedia.output.txt"]
+ANTI_RECOMMENDER_TYPE = "OpenAI"
+CI = true
```

Comment:
You shouldn't need to set this. GitHub Actions does it in the runner environment.

Reply:
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables